settings/defaults.php

```diff
@@ -85,32 +85,32 @@
 @define('CONST_Replication_Recheck_Interval', '60');
 
 // Website settings
-@define('CONST_NoAccessControl', true);
+// @define('CONST_NoAccessControl', true);
 
 @define('CONST_Website_BaseURL', 'http://'.php_uname('n').'/');
 // Language to assume when none is supplied with the query.
 // When set to false, the local language (i.e. the name tag without suffix)
 // will be used.
 @define('CONST_Default_Language', false);
 // Appearance of the map in the debug interface.
-@define('CONST_Default_Lat', 20.0);
-@define('CONST_Default_Lon', 0.0);
-@define('CONST_Default_Zoom', 2);
-@define('CONST_Map_Tile_URL', 'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png');
-@define('CONST_Map_Tile_Attribution', ''); // Set if tile source isn't osm.org
+// @define('CONST_Default_Lat', 20.0);
+// @define('CONST_Default_Lon', 0.0);
+// @define('CONST_Default_Zoom', 2);
+// @define('CONST_Map_Tile_URL', 'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png');
+// @define('CONST_Map_Tile_Attribution', ''); // Set if tile source isn't osm.org
 
 @define('CONST_Search_AreaPolygons', true);
 
-@define('CONST_Search_BatchMode', false);
+// @define('CONST_Search_BatchMode', false);
 
 @define('CONST_Search_NameOnlySearchFrequencyThreshold', 500);
 // If set to true, then reverse order of queries will be tried by default.
 // When set to false only selected languages alloow reverse search.
-@define('CONST_Search_ReversePlanForAll', true);
+// @define('CONST_Search_ReversePlanForAll', true);
 
 // Maximum number of OSM ids that may be queried at once
 // for the places endpoint.
-@define('CONST_Places_Max_ID_count', 50);
+// @define('CONST_Places_Max_ID_count', 50);
```

Comment:
BatchMode, ReversePlanForAll, Max_ID_count are part of the API features and not related to the website frontend. In the HTML frontend you can't use the features really. We added the limits to make sure end users can't overload the server with "expensive" queries.